### PR TITLE
Project detail popup

### PR DESCRIPTION
**Pull request by sangwa7**
### Project detail modal

1. Project detail modal popup layout
2. Display project detail content from JavaScript object

### CODE REVIEW

**Review by keroloussamy — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a linter error please fix it before requesting a review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

File: stle.css [hunk 1]
```diff
@@ -19,37 +19,38 @@ header {
...
+  color: white;
 }
 
 .close-icon {
   float: right;
   margin-right: 40px;
-  margin-top: 20px;
+  margin-top: 7px;
   width: 4%;
 }
```
Comment (thecodechaser) [inline stle.css]: - [ ] Good job working on the styles but still there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. Your design is a little different from the template. Please fix the close X button alignment for both desktop and mobile screens, Increase size of the button and make sure it matches the design.

Your design

Screenshot (191)


Figma design

Screenshot (190)

File: index.html [hunk 2]
```diff
@@ -108,22 +123,23 @@ <h3>I'm a software developer</h3>
...
           </div>
           <div class="line"></div>
         </div>
         <div class="section-2">
-          <div> <img src="img/Placeholder.svg" alt="" /> </div>
-          <div class="multi-post-stories"> 
-             <h2>Multi-Post Stories</h2>
+          <div><img src="img/Placeholder.svg" alt="" /></div>
+          <div class="multi-post-stories">
+            <h2>Multi-Post Stories</h2>
```
Comment (thecodechaser) [inline index.html:137]: - [ ] The pop-up model functionality is not working for your first two projects, Please make sure it works for all of your listed projects on the webpage.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @sangwa7👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a title and summary. ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (gemmen29) [inline index.html:137]: - Kindly fix the pop-up model functionality that is not working for your first two projects as the previous reviewer asked.
When I press the `see project` button in the first two projects the page is refreshed. The second project is worked but if I pressed the project itself not the `see project` button.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @sangwa7, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (FrederickMih) [inline index.html:137]: - [ ] Hi @sangwa7, you haven't still resolved this issue of the pop-up model functionalities pointed out by the previous reviewers for the first project. When the user clicks the **see project** button to check project details, the popup with details about the project should appear. Please kindly fix this issue and submit your project once more.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @sangwa7 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — COMMENTED** [review]
Hello @sangwa7  👋,

_This is an additional review as a result of the second opinion request._

The first project card "see project" button isn't popping up when clicked as pointed out by the code reviewers, this is a requirement for this project, kindly fix this and submit for another code review 
## Check the screenshot below for the faulty button 
image

Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀! //

File: index.html [hunk 3]
```diff
@@ -108,22 +123,23 @@ <h3>I'm a software developer</h3>
...
           <div class="title-h1">
             <h1>My Recent Works</h1>
           </div>
           <div class="line"></div>
         </div>
         <div class="section-2">
-          <div> <img src="img/Placeholder.svg" alt="" /> </div>
-          <div class="multi-post-stories"> 
-             <h2>Multi-Post Stories</h2>
+          <div><img src="img/Placeholder.svg" alt="" /></div>
```
Comment (odhiambo-ed) [inline index.html:135]: I recommend you add a description in the `Alt`  attribute to specifies an alternate text for an area, if the image cannot be displayed.

**Review by odhiambo-ed — COMMENTED** [review]
Hi @sangwa7 ,
You have been doing a tremendous job so far 👌
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors ✔️

Required Changes ♻️
 Please also check inline comments under the review
Optional suggestions
Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better.

Happy coding!👏👏👏
Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker reply or reach out to me on slack
Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Pull request merged**